pysat/__init__.py

```diff
@@ -71,33 +78,51 @@
         data_dir = '/home/travis/build/pysatData'
     else:
         data_dir = ''
-    with open(os.path.join(pysat_dir, 'data_path.txt'), 'w') as f:
+    data_path_file = os.path.join(pysat_dir, 'data_path.txt')
+    with Lock(data_path_file, 'w', file_timeout) as f:
         f.write(data_dir)
+        
+        # in case of network files system
+        f.flush()
+        os.fsync(f.fileno())
```

Comment:
```suggestion
    with Lock(data_path_file, 'w', file_timeout) as fout:
        fout.write(data_dir)
        
        # in case of network files system
        fout.flush()
        os.fsync(fout.fileno())
```

Reply:
Here and elsewhere to avoid single character non-iterator variable names.

Reply:
Implemented in #525